pom.xml

```diff
@@ -95,14 +95,14 @@
 
   <distributionManagement>
     <snapshotRepository>
-      <id>njord-snapshots</id>
-      <name>Local Snapshots Staging</name>
-      <url>njord:snapshot-sca</url>
+      <id>sonatype-cp</id>
```

Comment:
This id for Maven Central? I thought this ought to be `central`?

Reply:
No, the ID represents a publisher service (in form of `$provider-$url`) that we use, and it does publish to that given URL, which is central, yes.

Reply:
but that would mean id would change if a different publisher service is used in the future

Reply:
that means it goes into build info and should be removed from consumer pom